src/helpers/prepare-queued-pr-for-merge.ts

```diff
@@ -46,9 +46,11 @@ export const updatePrWithMainline = async (pullRequest: PullRequest | SimplePull
   } catch (error) {
     if ((error as GithubError).status === 204) {
       core.info('The first PR in the queue is already up to date!');
-    }
-    if ((error as GithubError).status === 409) {
-      core.info('The first PR in the queue has a merge conflict.');
+    } else if ((error as GithubError).status >= 400) {
+      if ((error as GithubError).status === 409) core.setFailed('The first PR in the queue has a merge conflict.');
+      else core.setFailed((error as GithubError).message);
```

Comment:
Cool, makes sense. So now I dont think this line is necessary since it's duplicating the generic error handler in `main.ts`